build_docs fails: cannot import name 'TestDumpParser'

**Describe the bug**

CI fails in **build_docs** with
```
Traceback (most recent call last):
  File "/home/runner/work/UserGuide/UserGuide/doc/source/scripts/gen_topologyparser_attrs.py", line 29, in <module>
    from MDAnalysisTests.topology.test_lammpsdata import LammpsBase, TestDumpParser
ImportError: cannot import name 'TestDumpParser' from 'MDAnalysisTests.topology.test_lammpsdata' (/home/runner/micromamba/envs/mda-user-guide/lib/python3.10/site-packages/MDAnalysisTests/topology/test_lammpsdata.py)
make: *** [Makefile:30: html] Error 1
make: Leaving directory '/home/runner/work/UserGuide/UserGuide/doc'
```

**To Reproduce**
Run CI

**Expected behavior**
CI should pass if I didn't specifically change the lammps tests.

**Actual behavior**
see error above and in https://github.com/MDAnalysis/UserGuide/actions/runs/14414547796/job/40428982295?pr=420



**Additional context**
See PR https://github.com/MDAnalysis/UserGuide/pull/420

--- a/doc/source/scripts/gen_topologyparser_attrs.py
+++ b/doc/source/scripts/gen_topologyparser_attrs.py
@@ -26,7 +26,10 @@ from MDAnalysisTests.topology.test_gms import GMSBase
 from MDAnalysisTests.topology.test_gro import TestGROParser
 from MDAnalysisTests.topology.test_gsd import TestGSDParser
 from MDAnalysisTests.topology.test_hoomdxml import TestHoomdXMLParser
-from MDAnalysisTests.topology.test_lammpsdata import LammpsBase, TestDumpParserFull
+from MDAnalysisTests.topology.test_lammpsdata import (
+    LammpsBase,
+    TestDumpParserFull,
+)
 from MDAnalysisTests.topology.test_mmtf import TestMMTFParser
 from MDAnalysisTests.topology.test_mol2 import TestMOL2Base
 from MDAnalysisTests.topology.test_pdb import TestPDBParser

--- a/doc/source/scripts/gen_topologyparser_attrs.py
+++ b/doc/source/scripts/gen_topologyparser_attrs.py
@@ -26,7 +26,7 @@ from MDAnalysisTests.topology.test_gms import GMSBase
 from MDAnalysisTests.topology.test_gro import TestGROParser
 from MDAnalysisTests.topology.test_gsd import TestGSDParser
 from MDAnalysisTests.topology.test_hoomdxml import TestHoomdXMLParser
-from MDAnalysisTests.topology.test_lammpsdata import LammpsBase, TestDumpParser
+from MDAnalysisTests.topology.test_lammpsdata import LammpsBase, TestDumpParserFull
 from MDAnalysisTests.topology.test_mmtf import TestMMTFParser
 from MDAnalysisTests.topology.test_mol2 import TestMOL2Base
 from MDAnalysisTests.topology.test_pdb import TestPDBParser
@@ -61,7 +61,7 @@ PARSER_TESTS = (
     TestTXYZParser,
     TestXPDBParser,
     XYZBase,
-    TestDumpParser,
+    TestDumpParserFull,
 )
 
 

